src/napari_spatialdata/_widgets.py

```diff
@@ -232,6 +240,65 @@ def _(self, vec: pd.Series, key: str, layer: Layer) -> dict[str, Any]:
         }
 
 
+class ScatterListWidget(AListWidget):
+    attrChanged = Signal()
+    _text = None
+
+    def __init__(self, viewer: Viewer, model: ImageModel, attr: str, **kwargs: Any):
+        AListWidget.__init__(self, viewer, model, attr, **kwargs)
+        self.attrChanged.connect(self._onChange)
+        self.data = -1
```

Comment:
what's this? don't set default unless really necessary, and if necessary set them meaningful or add comments.

Reply:
i will set as default to None. the idea was to use a default to check if anything has been selected before attempting to do the scatterplot